### PR TITLE
BOSE Support

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by tylerlange** [comment]
:+1:

File: devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy [hunk 1]
```diff
@@ -0,0 +1,989 @@
+/**
+ *  Bose SoundTouch
+ *
+ *  Copyright 2015 Henric Andersson
```
Comment (lbredeso) [inline devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy]: Do we individually copyright SmartApps?  I had thought copyright for SmartApps written by employees would be assigned to "SmartThings" or "SmartThings, Inc.".

Comment (workingmonk) [inline devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy]: Asked him to update that

**Comment by tylerlange** [comment]
re- :+1:

**Comment by workingmonk** [comment]
:+1:

**Pull request merged**